Dockerfile

```diff
@@ -25,4 +26,4 @@ RUN gem update bundler
 RUN bundle install
 
 # Run the app
-CMD bundle exec foreman start
+CMD cd $APP_HOME && bundle exec foreman start
```

Comment:
Why add the 'cd' here? Line 19 switches the context to the application home so you're already there when this command is run.